src/app-layer-htp-range.h

```diff
@@ -40,7 +40,7 @@ typedef struct HttpRangeContainerBuffer {
     /** offset of bytes written in buffer (relative to the start of the range) */
     uint64_t offset;
     /** number of gaped bytes */
-    uint64_t gap;
+    uint32_t gap;
```

Comment:
can we have >4GiB files, having gaps >4GiB as well?

Reply:
If a TCP stream has multiple gaps, whose addition is over 4GiB...

What do you think ? We should keep it u32 and saturate it ?